paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -406,6 +433,9 @@ void CPUQuantizePass::QuantizePriorBox(Graph* graph) const {
     // get scales calculated after warmup, they scale variables to MAX=1.0
     auto scales = Get<VarQuantScale>("quant_var_scales");
 
+    // make sure all the required scales are available
+    CheckScalesForVariables({prior_box_input->Name()});
```

Comment:
Same as above, but for PriorBox

Reply:
No, `prior_box` INT8 kernel does not return quantized output.

Reply:
I see, good to know. Thanks!